### PR TITLE
Test

### PR DESCRIPTION
**Pull request by akshaykumart**
Added code for removing users.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -17,3 +17,13 @@
     src: etc/sudoers.d/template
     dest: "/etc/sudoers.d/{{ item }}"
   loop: "{{ administrators_names }}"
+  
+- name: remove user
+  user:
+    name: "{{ item.name }}"
+    shell: "{{ administrators_default_shell }}"
+    state: absent
```
Comment (bngsudheer) [inline tasks/main.yml:26]: We also want to add the users only if they are `active`

File: vars/main.yml [hunk 2]
```diff
@@ -1,2 +1,20 @@
 ---
 # vars file for administrators
+---
```
Comment (bngsudheer) [inline vars/main.yml]: we don't want this file

**Review by bngsudheer — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 1 above)
Comment (akshaykumart) [reply tasks/main.yml:26]: done

File: vars/main.yml (hunk 2 above)
Comment (akshaykumart) [reply vars/main.yml]: done

File: README.md [hunk 3]
```diff
@@ -10,7 +10,13 @@ None
 Role Variables
 --------------
 ```yml
-administrators_names: ['admin01', 'admin02']
+administrators_names: 
+  -  name: "admin01"
+     active: true
+
```
Comment (bngsudheer) [inline README.md:16]: remove blank line in between list elements

File: README.md [hunk 4]
```diff
@@ -26,7 +32,13 @@ Example Playbook
 ```yml
   - hosts: servers  
     vars:
-      - administrators_names: ['admin01', 'admin02']
+      - administrators_names: 
+          -  name: "admin01"
+             active: true
+
```
Comment (bngsudheer) [inline README.md:38]: extra line in between list elements

File: README.md [hunk 5]
```diff
@@ -26,7 +32,13 @@ Example Playbook
...
+          -  name: "admin02"
+             active: true
+             
       - administrators_keys:
           - username: admin01
             key: /path/to/id_rsa_pub_admin01
```
Comment (bngsudheer) [inline README.md:44]: active: true is missing

File: tasks/main.yml [hunk 6]
```diff
@@ -17,3 +18,14 @@
     src: etc/sudoers.d/template
     dest: "/etc/sudoers.d/{{ item }}"
```
Comment (bngsudheer) [inline tasks/main.yml:19]: item.name

**Pull request closed without merging**